var/spack/repos/builtin/packages/py-torch-fidelity/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class PyTorchFidelity(PythonPackage):
+    """High-fidelity performance metrics for generative models in PyTorch"""
+
+    homepage = "https://www.github.com/toshas/torch-fidelity"
+    pypi = "torch_fidelity/torch_fidelity-0.3.0.tar.gz"
+
+    license("Apache-2.0", checked_by="qwertos")
+
+    version("0.3.0", sha256="3d3e33db98919759cc4f3f24cb27e1e74bdc7c905d90a780630e4e1c18492b66")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("pil", type=("build", "run"))
```

Comment:
I think you mean `py-pillow` here (according to the staged `requires.txt`):

```suggestion
    depends_on("py-pillow", type=("build", "run"))
```

Reply:
Is replacing all `py-pillow` dependencies with the provider `pil` not a thing anymore?

Reply:
I'm new to checking python package PR dependencies so wasn't aware it was a thing.  @adamjstewart ?

Reply:
Nvm.  I grep'd python packages and see `pil` is a virtual.